### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: server/viewModels/productSelectorViewModelBuilder.js [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+
+	barrierModel.packages.premium.subscribeLink = getEditedSubscribeLink(barrierModel.packages.premium.subscribeLink);
+
+	if(barrierModel.packages.newspaper) {
+		barrierModel.packages.newspaper.subscribeLink = getEditedSubscribeLink(barrierModel.packages.newspaper.subscribeLink);
+	}
+}
+
+function getEditedSubscribeLink(link) {
+	var newLink = link.replace('segid=70000', 'segID=400870');
```
Comment (aintgoin2goa) [inline server/viewModels/productSelectorViewModelBuilder.js]: You can't guarantee that will always be there - better to check for it first and then replace it or just add it if not

File: server/viewModels/productSelectorViewModelBuilder.js [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+	newLink += '&segmentID+42e7b7e1-3051-77ad-1677-daa49d96c2f1';
```
Comment (aintgoin2goa) [inline server/viewModels/productSelectorViewModelBuilder.js]: shouldn't this be &segmentID=42e7b7e1-3051-77ad-1677-daa49d96c2f1 ?

Comment (adamsilver) [inline server/viewModels/productSelectorViewModelBuilder.js]: Isn't that what it says?

File: server/viewModels/productSelectorViewModelBuilder.js (hunk 1 above)
Comment (adamsilver) [inline server/viewModels/productSelectorViewModelBuilder.js]: Okay will make defensive now

File: server/viewModels/productSelectorViewModelBuilder.js (hunk 2 above)
Comment (adamsilver) [inline server/viewModels/productSelectorViewModelBuilder.js]: Change the + to =

**Comment by aintgoin2goa** [comment]
:+1:

**Pull request merged**